cartographer_grpc/framework/server.h

```diff
@@ -81,6 +83,9 @@ class Server {
   // Shuts down the server and all of its services.
   void Shutdown();
 
+  // Sets the server-wide context object shared between RPC handlers.
+  void SetExecutionContext(std::unique_ptr<ExecutionContext> execution_context);
```

Comment:
Perhaps the server-wide context is better named ServerContext instead of ExecutionContext?

Reply:
The problem with that is that there is already a ::grpc::ServerContext that is store in every Rpc object. Would you prefer `HandlerContext`?